section1/reflection.md

```diff
@@ -1,19 +1,30 @@
 ## Section 1 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+* I feel that I'm okay asking questions in class (possibly too quickly), and then I tend not to ask questions much when I'm working alone.
 
 2. How would you print the string `"Hello World!"` to the terminal?
+* p "Hello World!" then run the ruby doc from the terminal.
 
 3. What character is used to indicate comments in a ruby file?
+* # (octothorpe)
 
 4. Explain the difference between an integer and a float?
+* Integer will always represent a whole number (1, 2, 3 humans), where as float numbers will use the decimal points (1.33 kilograms)
 
 5. In the space below, create a variable `animal` that holds the string `"zebra"`
+animal = "zebra"
 
 6. How would you print the string `"zebra"` using the variable that you created above?
+p "#{animal}"
 
 7. What is interpolation? Use interpolation to print a sentence using the variable `animal`.
+* Interpolation is when you use a variable to enter that data into a string or command in the terminal.
+p "Wow, would you look at that #{animal} go!"
 
 8. What method is used to get input from a user?
+* gets.chomp
 
 9. Name and describe two common string methods:
+* Use " " to surround what string you want printed.
+* Use .to_s for an integer to be printed as a string.
```

Comment:
I know you've made progress since section1! But looking back for your own benefit (no deliverable):

What 2 string methods would you submit now? If I clarified this as `string manipulation methods` would that change your answer?

Can you think of 2 String methods that we'd use to manipulate strings?